tree-sitter-darklang/grammar.js

```diff
@@ -695,7 +697,16 @@ module.exports = grammar({
       prec(
         PREC.FIELDACCESS,
         seq(
-          field("expr", $.expression),
+          field(
+            "expr",
+            choice(
+              $.variable_identifier,
+              $.record_literal,
+              $.paren_expression,
+              $.field_access,
+              $.apply,
+            ),
+          ),
```

Comment:
is this update a bad idea?
    this change was because `1L |> Stdlib.Int64.toString` was parsed as a
    fieldAccess of a pipeEnum i.e `(1L |> Stdlib.Int64).toString` instead of a pipeFnCall `1L |> Stdlib.Int64.toString`

Reply:
I think it's actually a great solution - clever. Maybe "clever" will end up being _too clever_ (it happens sometimes) but it seems reasonable to me. Luckily, there's composition in play here, so we can always adjust this later if that turns out to be the case.

Anyway, I'd keep it!